docker/node/server.js

```diff
@@ -548,10 +564,79 @@ function isUserBanned(o) {
 
 	return false;
 }
+function augmentBan(ban, o) {
```

Comment:
Why is this function touched in this PR? Seems to just be copied (not moved) from server.js?